uns_plugin/main.go

```
@@ -0,0 +1,21 @@
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package uns_plugin
+
+import "github.com/redpanda-data/benthos/v4/public/service"
+
+func init() {
+	service.RegisterBatchOutput("uns", outputConfig(), newUnsOutput)
```

golangci-lint (1.64.8)

Error return value of `service.RegisterBatchOutput` is not checked

(errcheck)